model/src/test/test_model_hospitalizations.py

```diff
@@ -361,7 +377,7 @@ def test_model_hosp_with_obs_model():
 
     # Sampling and fitting model 0 (with no obs for infections)
     np.random.seed(223)
-    with npro.handlers.seed(rng_seed=np.random.randint(1, 600)):
+    with numpyro.handlers.seed(rng_seed=np.random.randint(1, 600)):
```

Comment:
not from this PR, but I think we should do away with this pattern. It is just as "random" as if we did with `numpyro.handlers.seed(rng_seed=223):`

Reply:
I agree. I removed it in a bunch of other places in this PR and the last.

Reply:
Addressed in 1081ae2